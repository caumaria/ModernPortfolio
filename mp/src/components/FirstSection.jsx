import Header from './Header'
import styled from 'styled-components';
import MainSection from './MainSection';
import SocialBar from './SocialBar';

const FirstSectionContainer = styled.div`
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    height: 100vh;
`;

const Conteiner = styled.div`
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
`;

export default function FirstSection() {
  return (
    <FirstSectionContainer>
      <Header />
        <Conteiner>
          <MainSection />
        </Conteiner>
        <SocialBar />
    </FirstSectionContainer>
  )
}
