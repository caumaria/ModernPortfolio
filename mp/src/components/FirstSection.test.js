import "@testing-library/jest-dom";
import renderer from 'react-test-renderer';
import FirstSection from "./FirstSection";
import React from 'react'; // eslint-disable-line no-unused-vars

describe("should render hero component correctly", () => {
    it('performs snapshot testing', () => {
      expect.hasAssertions();
      const tree = renderer.create(<FirstSection />).toJSON();
      expect(tree).toMatchSnapshot();
    });
});