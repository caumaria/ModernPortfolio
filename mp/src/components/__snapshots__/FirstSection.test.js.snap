// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render hero component correctly performs snapshot testing 1`] = `
<div
  className="sc-hHftZz gCRUBB"
  id="home"
>
  <div
    className="sc-dmlpXa jhMwDg"
  >
    <div
      className="sc-dlfmHC"
      style={{}}
    >
      <div
        className="sc-hKgKIp"
        style={
          {
            "opacity": 0,
            "transform": "translateX(-100px) translateY(0px) translateZ(0)",
          }
        }
      >
        <div
          className="sc-eCstZk bLthyG"
        >
          <div
            className="sc-jSguLX isflGR"
          >
            <div />
          </div>
          <div
            className="sc-bdfCDU hnyfnA"
          >
            <div
              className="sc-gsTDqH cFswNq"
            >
              <li
                data-content="Home"
                onClick={[Function]}
              >
                Home
              </li>
              <a
                onClick={[Function]}
                style={{}}
              >
                <li
                  data-content="About Me"
                >
                  About Me
                </li>
              </a>
              <a
                onClick={[Function]}
                style={{}}
              >
                <li
                  data-content="Work"
                >
                  Work
                </li>
              </a>
              <a
                onClick={[Function]}
                style={{}}
              >
                <li
                  data-content="Contact"
                >
                  Contact
                </li>
              </a>
            </div>
          </div>
          <div
            className="sc-gKsecS eAcDxx"
          >
            <div />
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-kfzCjt dCgysv"
    >
      <div
        className="sc-jrAGKZ cPBvYX"
        style={{}}
      >
        <div
          className="sc-kEjbdu kJYzal"
          style={
            {
              "opacity": 0,
              "transform": "translateX(0px) translateY(-100px) translateZ(0)",
            }
          }
        >
          <img
            alt="Photo of Cau."
            src=""
          />
          <div
            className="sc-iBPTVF hOBOTs"
          >
            <p>
              Cau\`s Portfolio, software engineer. A driven and dynamic individual with a goal to create exceptional web & apps experiences. Let\`s work together to make a lasting impact. Tech Tags: React, JavaScript, Typescript, NextJS.
            </p>
          </div>
        </div>
        <div
          className="sc-iqHXzD gjGkyk"
          style={
            {
              "opacity": 0,
              "transform": "translateX(0px) translateY(-100px) translateZ(0)",
            }
          }
        >
          <img
            alt="Photo of Cau."
            src=""
          />
          <section>
            <p>
              React
            </p>
            <p>
              Javascript
            </p>
            <p>
              Typescript
            </p>
            <p>
              CSS
            </p>
            <p>
              NextJS
            </p>
            <p>
              NodeJs
            </p>
          </section>
          <p>
            A driven and dynamic individual with a goal to create exceptional web & apps experiences. Let\`s work together to make a lasting impact.
          </p>
        </div>
        <div
          className="sc-fubDmA fdfYXG"
          style={
            {
              "opacity": 0,
              "transform": "translateX(0px) translateY(100px) translateZ(0)",
            }
          }
        >
          <div>
            <a
              onClick={[Function]}
              style={{}}
            >
              <button>
                Let 
                <span>
                  â€™
                </span>
                 s get in touch
                <span>
                  !
                </span>
                <img
                  alt="Arrow pointing to ways of contact"
                  src=""
                />
              </button>
            </a>
            <div
              className="sc-pGaPU kqdzQK"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-hBEZvw dGlwES"
      style={{}}
    >
      <div
        className="sc-fodVRF"
        style={
          {
            "opacity": 0,
            "transform": "translateX(100px) translateY(0px) translateZ(0)",
          }
        }
      >
        <div
          className="sc-eCstZk bLthyG"
        >
          <div
            className="sc-fFubCH gUhBXD"
          >
            <div
              className="sc-bkzXAi bCeSCc"
            >
              <div />
            </div>
          </div>
          <div
            className="sc-crrtmM hYNtpV"
          >
            <div
              className="sc-dQpIV hXcavR"
            >
              <a
                className="sc-bqyKbq llzPEO"
                href="https://github.com/caumaria"
                target="blank"
              >
                <img
                  alt="Github de Cau"
                  src=""
                />
              </a>
              <div
                className="sc-kstpWv bMeGUc"
              >
                <img
                  alt="Me siga nas redes sociais:"
                  src=""
                />
              </div>
              <a
                className="sc-bqyKbq llzPEO"
                href="https://www.linkedin.com/in/cau-ugolini-923a4a185/"
                target="blank"
              >
                <img
                  alt="Linked In de Cau"
                  src=""
                />
              </a>
              <div
                className="sc-kstpWv bMeGUc"
              >
                <img
                  alt="Me siga nas redes sociais:"
                  src=""
                />
              </div>
            </div>
          </div>
          <div
            className="sc-idOjMB bIUszM"
          >
            <div
              className="sc-dIUfKc dexbKx"
            >
              <div />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
